src/send/EnterAmountOptions.tsx

```diff
@@ -90,7 +93,7 @@ export default function EnterAmountOptions({
         styles.container,
         {
           position: 'absolute',
-          bottom: 0,
+          bottom: demoModeEnabled ? -variables.demoModeBorderWidth : 0,
```

Comment:
the enter amount percentages are positioned relative to the bottom of the navigation container, which has a border in demo mode, but the keyboard is always positioned relative to the bottom of the screen. without accounting for the border, there is an awkward gap between the percentages and the keyboard :(